src/react-components/chat-message.js

```diff
@@ -4,12 +4,10 @@ import PropTypes from "prop-types";
 import styles from "../assets/stylesheets/presence-log.scss";
 import classNames from "classnames";
 import { toArray as toEmojis } from "react-emoji-render";
-import serializeElement from "../utils/serialize-element";
+import html2canvas from "html2canvas";
```

Comment:
I don't have a VR headset handy to test perf... (you mentioned in your comments might be a problem)

But it seems good that `html2canvas` only pulls in a single dependency.